docs/google.rst

```diff
@@ -17,12 +19,74 @@ Google BigQuery is a cloud data warehouse solution. Data is stored in tables, an
 BigQuery uses datasets as top level containers for tables, and datasets are themselves contained within
 Google Cloud projects.
 
+==========
+Quickstart
+==========
+
+To instantiate the GoogleBigQuery class, you can pass the constructor a string containing either the name of the Google Drive credentials file or a JSON string encoding those credentials. Alternatively, you can set the environment variable ``GOOGLE_APPLICATION_CREDENTIALS`` to be either of those strings and call the constructor without that argument.
```

Comment:
I think "Google service account credentials file" is better than "Google Drive credentials file".

Reply:
Fixed! Good catch.